pydbus/strong_typing.py

```diff
@@ -34,3 +35,10 @@ def decorate(func):
 		verify_arguments(func)
 		return func
 	return decorate
+
+
+class TypedSignal(signal):
```

Comment:
That's a good question. Following the builtins's and functools's convention, it should. While I feel with `_` it's more readable, we shouldn't break the rule.

So yeah, the correct names are `typedproperty`, `typedmethod` and `typedsignal`.